pkgs/applications/virtualization/singularity/default.nix

```diff
@@ -46,24 +49,23 @@ buildGoPackage rec {
 
     # Don't install SUID binaries
     sed -i 's/-m 4755/-m 755/g' builddir/Makefile
-
   '';
 
   buildPhase = ''
+    runHook preBuild
     make -C builddir
+    runHook postBuild
   '';
 
   installPhase = ''
+    runHook preInstall
     make -C builddir install LOCALSTATEDIR=$out/var
     chmod 755 $out/libexec/singularity/bin/starter-suid
-    wrapProgram $out/bin/singularity --prefix PATH : ${stdenv.lib.makeBinPath propagatedBuildInputs}
+    runHook postInstall
   '';
 
   postFixup = ''
     find $out/libexec/ -type f -executable -exec remove-references-to -t ${go} '{}' + || true
```

Comment:
Different PR, but should we include $out/libexec in buildGoPackage as well for removing references?

Reply:
I think it would be valuable to do so, libexec is part of the FHS so we should expect some binaries to land there.